src/components/ColumnsDisplay/index.tsx

```diff
@@ -169,22 +177,13 @@ const styles = {
   checked: {},
 };
 
-interface SelectedColumnsData {
-  cardinality: null | number;
-  dataType: string;
-  displayName: string;
-  entity: string;
-  keyType: string;
-  name: string;
-}
-
 type Props = {
-  selectedColumns: any;
+  selectedColumns: any[];
   selectedEntity: string;
   attributes: any;
   classes: any;
   submitValues: () => void;
-  setColumns: (columns: object[]) => void;
+  setColumns: (entity: string, columns: object[]) => void;
```

Comment:
If we're all in on Typescript, should all `object` become `any`?

Reply:
@anonymoussprocket we need to use the correct type.

Reply:
This is for later:

https://github.com/Cryptonomic/Arronax/issues/158